packages/core/src/git.ts

```diff
@@ -488,6 +491,7 @@ export default class Git {
   }
 
   /** Search to GitHub project's issue and pull requests */
+  @memoize()
```

Comment:
In a single run if the search query is the same it should be safe to memoize. The `search` endpoints have super low rate limits so this should help a bit.